org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/index/ResourceDescriptionStrategyFragment.xtend

```diff
@@ -122,6 +126,9 @@ class ResourceDescriptionStrategyFragment extends AbstractInheritingFragment {
 						return createEObjectDescriptionsFor�exportedRule.type.classifier.name�(eObject, acceptor);
 					}
 				�ENDFOR�
+				�IF needsToInvokeSuperResourceDescriptionStrategy�
+				super.createEObjectDescriptions(eObject, acceptor);
```

Comment:
Should be `return super.createEObjectDescriptions(..)` instead of `return true` in that case